awx/main/tasks.py

```diff
@@ -2895,6 +2895,13 @@ def __init__(self, task=None, runner_params=None):
         if self.task and not self.task.instance.is_container_group_task:
             execution_environment_params = self.task.build_execution_environment_params(self.task.instance, runner_params['private_data_dir'])
             self.runner_params['settings'].update(execution_environment_params)
+        elif self.task and self.task.instance.is_container_group_task:
+            pass
+            # look to see if there is a credential on the EE
+            # use the kubernetes python lib to create  an image pull secret
+            # inject `imagePullSecret` into pod definition.  maybe do this inside the pod_definition(),
+            #   or in a function called by pod_definition, which will already only be called when.
+            #   will be a list of diction
```

Comment:
Is this still a work in progress, or is it accomplished later in the diff here?

Reply:
That can be removed now, I'll do that now, thanks.